Converters/Civil3d/Speckle.Converters.Civil3dShared/ToSpeckle/Properties/PropertySetDefinitionHandler.cs

```diff
@@ -53,7 +51,7 @@ AAECPDB.PropertySetDefinition setDefinition
 
     if (Definitions.ContainsKey(name))
     {
-      return (propertyDefinitionNames, name);
+      return (propertyDefinitionNames);
```

Comment:
is that parenthesis redundant? or are we returning a tuple of 1 item here? I'm assuming it's the former :)